index.html

```diff
@@ -220,6 +220,32 @@ <h3 class="small__resume_title">Skills</h3>
         </section>
 
         <!-- Contact Form -->
+        <section class="contacts">
```

Comment:
- [ ] Hello @mfadhili  , Well done on the changes. Sadly, there is still a matter that has to be addressed, but you are almost there. As instructed in the last review, you should use the correct `background` so that your design will match the Figma design, kindly fix this :wink:  :upside_down_face: 

This how your Design still looking like :point_down: 
![formsreep](https://user-images.githubusercontent.com/88894525/194314743-4ef9fec7-ecef-4ceb-8a74-3c3314bea145.png)

Figma Design :point_down: 
![figmaformonedesi](https://user-images.githubusercontent.com/88894525/194314922-2b1e6c6e-f2ae-471c-831c-086cf671725a.png)